sklearn/metrics/_regression.py

```diff
@@ -574,7 +688,78 @@ def mean_squared_log_error(
         np.log1p(y_pred),
         sample_weight=sample_weight,
         multioutput=multioutput,
-        squared=squared,
+    )
+
+
+@validate_params(
+    {
+        "y_true": ["array-like"],
+        "y_pred": ["array-like"],
+        "sample_weight": ["array-like", None],
+        "multioutput": [StrOptions({"raw_values", "uniform_average"}), "array-like"],
+    },
+    prefer_skip_nested_validation=True,
+)
+def root_mean_squared_log_error(
+    y_true, y_pred, *, sample_weight=None, multioutput="uniform_average"
+):
+    """Root mean squared logarithmic error regression loss.
+
+
```

Comment:
This should fix the numpydoc error
```suggestion

```

Reply:
Don't you have a pre-commit check to autocorrect this?

Reply:
not for numpydoc,we run it as part of the test suite